packages/react/src/components/ComposedStructuredList/ComposedStructuredList.jsx

```diff
@@ -80,26 +36,38 @@ const StructuredList = ({
     );
   }
   return (
-    <Fragment>
-      <StructuredListWrapperStyled
+    <>
+      <StructuredListWrapper
         {...StructuredListWrapperProps}
+        className={classnames({
+          [`${iotPrefix}--composed-structured-list__wrapper`]: isFixedWidth,
```

Comment:
For future implementations, it would probably be better for these one-off class conditions to be set with css variables.